src/TestFramework/MSTest.Core/Assertions/StringAssert.cs

```diff
@@ -107,18 +158,21 @@ public static void Contains(string value, string substring, string message)
         /// is not in <paramref name="value"/>. The message is shown in
         /// test results.
         /// </param>
+        /// <param name="comparisonType">
+        /// The comparison method to compare strings <paramref name="comparisonType"/>.
+        /// </param>
         /// <param name="parameters">
         /// An array of parameters to use when formatting <paramref name="message"/>.
         /// </param>
         /// <exception cref="AssertFailedException">
         /// Thrown if <paramref name="substring"/> is not found in
         /// <paramref name="value"/>.
         /// </exception>
-        public static void Contains(string value, string substring, string message, params object[] parameters)
```

Comment:
please note, `Contains(string value, string substring, string message, params object[] parameters)` is now gone, so this is breaking change. 

Please also add  `Contains(string value, string substring, string message, params object[] parameters)` back

Reply:
Added